contracts/src/DeployScript.sol

```diff
@@ -59,9 +59,11 @@ contract DeployScript is Script {
 
         uint8 foreignTokenDecimals = uint8(vm.envUint("FOREIGN_TOKEN_DECIMALS"));
 
+        address recoveryOperator = vm.envOr("RECOVERY_OPERATOR", address(0));
+
```

Comment:
Maybe nitpicking, what's the benefit of using `address(0)` as a fallback instead of explicitly `vm.envAddress`?

Reply:
There is no fallback for `vm.envAddress` which requires the env to already explicitly be set, does that make more sense here?

Reply:
This is just a placeholder until we set up multisig to act as a recovery account.